### PR TITLE
Adds A2 Trial License if none provided

### PR DESCRIPTION
**Pull request by ghost**
Signed-off-by: Ruairi Fennell <rfennell@chef.io>

### Description

Currently we ask the user to provide an A2 license in the Azure UI during setup. This change makes adding a License optional. If no license is added, a Trial License will be applied.

Without License:
screen shot 2018-10-30 at 13 53 52

With License:
screen shot 2018-10-30 at 13 54 34

### CODE REVIEW

File: src/createUiDefinition.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
                 "toolTip": "A prefix that should be applied to all resources that are created",
                 "constraints": {
                     "required": true,
-                    "regex": "^[a-z0-9]{1,5}$",
+                    "regex": "^[a-z][a-z0-9]{1,4}$",
```
Comment (ghost) [inline src/createUiDefinition.json:14]: Removes ability for user to start Prefix with an integer.

**Review by russellseymour — APPROVED** [review]
Looks good :+1:

**Comment by trevorghess** [comment]
@r-fennell does that GDPR flag get presented on the UI?

**Comment by ghost** [comment]
@trevorghess Yes


screen shot 2018-11-26 at 12 06 41

**Pull request merged**